docs/source/open-edx-background.rst

```diff
@@ -0,0 +1,36 @@
+Open edX Background
+===================
+
+What is Open edX?
+--------------------
+Open edX refers to its two components: the Open edX Platform and the Open edX Community.
+The Open edX Platform is a scaleable and learner centered web-based platform that can be used to author and manage courses.
+This open source project is supported by an international community working together to create this multiuse platform.
+The Open edX Community consists of tCRIL, the nonprofit organization that runs the project, and a number of core and individual contributors. 
+For more information, see:
+
+- `What is "Open edX"? <https://open.edx.org/video/what-is-open-edx/>`_
+- `What makes the Open edX Platform unique? <https://open.edx.org/blog/what-makes-the-open-edx-platform-unique/>`_
+- `State of Open edX Community and Architecture <https://www.youtube.com/watch?v=HPrMYdwLsdE>`_
+
+More About the Open edX Community
+---------------------------------
+The Open edX community is international and includes those writing code to
+further the Open edX platform's capabilities, those authoring courses on a wide
+variety of Open edX instances, operators of those instances, instructional
+designers, translators, and more. The community is furthering the Open edX
+mission and expanding the reach and impact via numerous `working groups
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/46793351/Working+groups>`_
+and through the growing `core contributor program
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/3143205354/Open+edX+Core+Contributor+Program>`_.
```

Comment:
In the future this will be an OEP that we should probably point to instead of a wiki page. No action needed, just a note for us to remember.